website/registrations/models.py

```diff
@@ -140,9 +137,25 @@ class Meta:
         )
 
 
-class Registration(Entry):
+class Registration(Entry, Payable):
```

Comment:
The actual Payable object here would be the `Entry` class. Then you can override any other fields like you are doing in `Renewal` already.

Reply:
Shouldn't Entry be an ABC?

Reply:
No, both because it is a Django model and that is impossible. And it is already an abstract base model.